chart/kubeapps/templates/apprepositories-secret.yaml

```diff
@@ -0,0 +1,23 @@
+{{- range .Values.apprepository.initialRepos }}
+{{- if or .caCert .authorizationHeader }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "kubeapps.fullname" $ }}-{{ .name }}
```

Comment:
The name we create for the Secrets follows the format: `apprepo- {{ .name }}-secrets`. Please be consistent with that format: https://github.com/kubeapps/kubeapps/blob/4b21ca69f9c5817815f793b8ff6438128145c148/dashboard/src/actions/repos.ts#L138